Resources/Prototypes/_Arc/Entities/Clothing/Head/helmets.yml

```
@@ -0,0 +1,42 @@
+- type: entity
+
+  id: ClothingHeadHelmetFlesh
+  name: Flesh cult helmet
```

```suggestion
  name: flesh cult helmet
```
the title case in question: